lib/config_test.go

```diff
@@ -271,13 +271,31 @@ var _ = t.Describe("Config", func() {
 			Expect(err).To(BeNil())
 		})
 
-		It("should fail on invalid NetworkDir", func() {
+		It("should create the  NetworkDir", func() {
 			// Given
-			sut.NetworkConfig.NetworkDir = invalidPath
+			tmpDir := path.Join(os.TempDir(), invalidPath)
+			sut.NetworkConfig.NetworkDir = tmpDir
 
 			// When
 			err := sut.NetworkConfig.Validate(true)
 
+			// Then
+			Expect(err).To(BeNil())
+			os.RemoveAll(tmpDir)
+		})
+
+		It("should fail on invalid NetworkDir", func() {
+			// Given
+			tmpfile := path.Join(os.TempDir(), "wrong-file")
```

Comment:
Just a single non-blocking nit: We could use the `TestFramework` method `MustTempDir()` here, which should do the same job.